source/streaming-mode/streaming-read-config.txt

```diff
@@ -46,6 +46,27 @@ You can configure the following properties when reading data from MongoDB in str
    * - ``collection``
      - | **Required.**
        | The collection name configuration.
+       | You can specify multiple collections by separating the collection names
+         with a comma. The collections must be in the same database. Do not add
```

Comment:
```suggestion
         with a comma. Do not add
```

This sentence brings only confusion: there is no way to configure the connector to read from different databases (see the `database` configuration option), and the `collection` configuration option allows users to specify only the collection names, not namespaces (they include the database name). So this sentence rises questions like "How could I possibly specify collections from multiple databases?", "Does this mean there is a way to read from multiple databased via the connector?".

Reply:
Removed